src/middlewares/add-last-modified-at.ts

```diff
@@ -0,0 +1,8 @@
+import { WithLastModified } from "../types";
+
+export function addLastModified<T>(item: T): WithLastModified<T> {
+    return {
+        ...item,
+        _last_modified: new Date()
```

Comment:
Imo this should definitely be string and not date, e.g. new Date().toISOString() so it includes the timezone offset